domains/nils.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "TheNoim",
+    "email": "nilsbergmann@noim.io"
+  },
+  "description": "Use as alias for noim.io (My private webpage)",
+  "record": {
+    "URL": "https://noim.io"
```

Comment:
We are currently working on a bug related to url redirections #221. This may take a week to fix. I'd suggest using a cname record instead for now. If you want to keep it as it is, let me know, I'll merge it in.

Reply:
I can wait for the actual fix :)

Reply:
Alright. You can subscribe to #221 to get updates on the issue.